blocks/bricks/base.py

```diff
@@ -556,6 +582,7 @@ def __init__(self, name=None):
         self.name = name
 
         self.children = []
```

Comment:
It's to initialize the list, otherwise `self.children` would result in an `AttributeError` instead of returning an empty list. There's a dozen way to do this though, I could also do a `if not hasattr(self, '_children'): self.children = []` in the getter, but this was one line shorter :p